tests/unit/services/page-title-test.js

```diff
@@ -0,0 +1,37 @@
+import { settled } from '@ember/test-helpers';
```

Comment:
Suggestions for any additional tests?

Reply:
Maybe we should have acceptance test instead, where observe hook call by updating some DOM element?

Reply:
> by updating some DOM element
Do you mean using the `{{page-title}}` helper?

Reply:
Correct. Easiest is probably to add new route, we have few dummy routes for testing different scenarios. We can add new one for testing this one.

And in acceptance test we just register overridden service:page-title via `this.owner.register()` to capture "titleDidUpdate()" being called and we visit route X.

Reply:
For now, I just added the acceptance test to the existing `acceptance/posts-test.js` file. All current routes are tested using that file, so it seemed a bit weird to add a new file for this, but feel free to let me know if you want it tested somewhere else.